### PR TITLE
Avoids failing specs on unthemed components

### PR DESCRIPTION
**Pull request by dzucconi**
This should get the Force integration PRs (https://github.com/artsy/force/pull/7115) to pass

### CODE REVIEW

File: packages/palette/src/elements/Button/v3/Button.tsx [hunk 1]
```diff
@@ -116,7 +117,7 @@ const Container = styled.button<ContainerProps>`
   ${(props) => {
     // Handle pseudo classes
     return css`
-      @media ${props.theme.mediaQueries.hover} {
+      @media ${MEDIA_QUERY_HOVER} {
```
Comment (dzucconi) [inline packages/palette/src/elements/Button/v3/Button.tsx:120]: I guess actually I could just import this from each respective theme. I had thought this was weird initially when they were the same component.

File: packages/palette/src/Theme.tsx [hunk 2]
```diff
@@ -49,6 +49,12 @@ export const useTheme = <T extends TTheme>() => {
   return { theme }
 }
 
+/** Returns a config specific to the current theme */
+export const useThemeConfig = <T, U>({ v2, v3 }: { v2: T; v3: U }) => {
+  const { theme = { id: "v2" } } = useTheme()
+  return { v2, v3 }[theme.id as "v2" | "v3"]
```
Comment (damassi) [inline packages/palette/src/Theme.tsx:55]: #nit Can we make this form a little more conventional

Comment (dzucconi) [reply packages/palette/src/Theme.tsx:55]: WHAT HAPPENS WHEN WE HAVE THREE OR FOUR THEMES?

(I kid, will make it a ternary)

**Comment by artsyit** [comment]
:rocket: PR was released in [`@artsy/palette@13.36.1`](https://github.com/artsy/palette/releases/tag/%40artsy/palette%4013.36.1) :rocket:

**Pull request merged**